### ISSUE TITLE
There is no camtraptor.Rproj file included in the repo

### ISSUE THREAD

**Issue by PietrH** [inbo/camtraptor#182]
Should there be an Rproject file included in this repo? [ETN](https://github.com/inbo/etn) for example includes one. 

*.Rproj is a typical entry in .RBuildignore according to: https://r-pkgs.org/Structure.html, I'm unsure as to the rationale for including or excluding it

**Comment by PietrH** [comment]
Indeed, the Rproj files are included in RBuildignore since 95ca98d6d138885f3482523903c5d575c23f517e

**Comment by damianooldoni** [comment]
Thanks @PietrH to ask this. Indeed, based on https://r-pkgs.org/structure.html#sec-rbuildignore the R Studio project files should not be part of the repository. I agree to be consistent thorughout all packages we maintain/develop.

**Comment by peterdesmet** [comment]
I find it very useful though to have a Rproj file in our repos, so I would not ignore it. I notice it is also included for:

https://github.com/r-lib/pkgdown/blob/main/pkgdown.Rproj
https://github.com/tidyverse/dplyr/blob/main/dplyr.Rproj

**Comment by PietrH** [comment]
If you (and/or others) find the inclusion of these useful, I have no objection to keeping them.

Just as a small note; as I understand [from this page](https://r-pkgs.org/structure.html#sec-rbuildignore) these files can not be submitted to CRAN.

**Comment by peterdesmet** [comment]
Not submitted to CRAN: can be solved by including them in `.Rbuildignore`.

**Comment by PietrH** [comment]
Fair enough! I've learned something new. 

Instead of removing rproj from etn, instead we could add one here then

**Comment by peterdesmet** [comment]
Yep!

**Comment by damianooldoni** [comment]
Still, I see that many tidyverse packages have such RProj file in heir repo. tidyr and purrr also. I will add Rproj too

By the way, I don't find really useful to have a RProj in a R package repo. Just open RStudio -> New Proejct - > Existing directory and choosing the root directory of the package and voila. 

@peterdesmet: why do you find it useful? To avoid the steps I mentioned above?

Threre is still an inconsistency between what Hadly Wickham proposed in its book and the practice in tidyverse packages.

**Comment by damianooldoni** [comment]
Ok, @peterdesmet. I will fix this together with #185 tomorrow morning or even this evening.

**Comment by damianooldoni** [comment]
Actually we were speaking of two different things. 

The `.Rproj` files are in the **`.Rbuildignore`** of the tidyverse packages. But they are NOT in the **`.gitignore`**. So, the theory in the book of Wickham and the practice in the tidyverse pakcages are not contradicting each other. The links of @PietrH and I were speaking about the `.Rbuildignore`, which has nothing to do with the `.gitignore` of course. Citing the same book, I see that the only R project related entry that should be added to a `gitignore` is any `.Rproj.user` files or subdirectory with such name. See https://r-pkgs.org/whole-game.html#create_package.

In other words:
- YES to have `^.*\.Rproj$` in `.RBuildignore`
- NO to have `*.Rproj` in `.gitignore` 

Done in https://github.com/inbo/camtraptor/commit/2f7c470f0967b1b953e9ccf11a9b9304ca0e6373

**Comment by damianooldoni** [comment]
Solved via #186
